docs/guides/upgrading-cluster.md

```diff
@@ -16,16 +16,18 @@ You can update or upgrade your cluster using Terraform.
 
 ## Upgrading your cluster
 
-To upgrade your ROSA cluster to another version, export the following variable then run `terraform apply`.
+To upgrade your ROSA cluster to another version, upgrade your account roles and policies, export the following variables, and then run `terraform apply`.
+
+1. If you are upgrading from to a new Y-stream, such as upgrading from version 4.12.19 to 4.13.1, run `terraform apply` in the [the `account_roles` module again](../../examples/create_account_roles/README.md). You must specify the y-stream version that you wish to update to with the following variable. 
 
 1. Export the `TF_VAR_openshift_version` with the intended version. Your value must be prepended with `openshift-v`.
-    ```
-    export TF_VAR_openshift_version=<version_number>
-    ```
-1. Upgrading to a new cluster version, especially when transitioning between major y-streams, requires approval. You might be requested to provide administrative confirmation regarding significant modifications for your cluster. In this case, when you first attempt the upgrade, you will receive an error message that provides guidance on the necessary modifications. It is essential to follow those instructions carefully and indicate completion of the requirements by adding the "upgrade_acknowledgements_for" attribute to your resource, specifying the target version. For example, if you are upgrading from 4.11.43 to 4.12.21, you should use '4.12' as the value for this variable.
-    ```
-    upgrade_acknowledgements_for = <version_acknowledgement>
-    ```
+        ```
+        export TF_VAR_openshift_version=<version_number>
+        ```
+1. Upgrading your cluster requires approval, especially when transitioning between major Y-streams. You may be required to provide administrative confirmation regarding significant modifications for your cluster. In this case, when you first attempt the upgrade, you will receive an error message that provides guidance on the necessary modifications. It is essential to follow those instructions carefully and indicate completion of the requirements by adding the "upgrade_acknowledgements_for" attribute to your resource with your the target version. For example, if you are upgrading from 4.11.43 to 4.12.21, you should use '4.12' as the value for this variable.
```

Comment:
1. I don't think that "y-streams" should be capitalized.
2. I think it should read "significant modifications to your cluster" rather than "for", but it depends on context.
3. Again, not sure about the context, but it might read better as "an error message that provides guidance regarding the necessary modifications". This totally depends on who is doing the modifying, so it might be better as is.
4. I would change "those instructions" to "the instructions" because it is more specific and will also translate better.
5. I would break up the sentence that begin with "It is essential..." Have it read: "It is essential to follow the instructions carefully. Indication completion of the requirements...".
6. The second part of the sentence might have an extra word. It currently reads: "attribute to your resource with your the target version". I think the "the" before "target version" should be removed.
7. Maybe put "version" in front of "4.11.43" so that it matches how you compare two versions in line 21.